tests/solvers/test_radau5.py

```diff
@@ -794,19 +803,78 @@ def f(t, y):
         y0 = N.array([1.])
         prob = Explicit_Problem(f, y0)
         sim = Radau5ODE(prob)
-        sim.solver = 'f'
+        sim.implementation = 'f'
+
+        err_msg = f'Repeated unexpected step rejections.'
```

Comment:
You are using an f-string here but not formatting the string, I am just adding the comment here in case you missed something.